cmd/ctr/commands/images/tag.go

```diff
@@ -60,6 +65,9 @@ var tagCommand = cli.Command{
 
 		if !context.BoolT("local") {
 			for _, targetRef := range context.Args()[1:] {
+				if _, err := reference.ParseAnyReference(targetRef); !context.Bool("skip-reference-check") && err != nil {
```

Comment:
```suggestion
				if !context.Bool("skip-reference-check") {
					if _, err := reference.ParseAnyReference(targetRef); err != nil {
```

This has more nesting, but raises the flag in terms of prominence and allows us to skip a call to `ParseAnyReference` if it's not being used.  I think it's easier to read than the composite conditional embedded in the same line.

Reply:
Done.